packages/assertions/assertions.0.1/descr

```diff
@@ -0,0 +1,8 @@
+Short
```

Comment:
You can remove `Short description` and `Long description` from the file. The first line will be consider as opam as the short description, the rest will be the long description.